paddle/fluid/operators/run_program_op.h

```diff
@@ -251,6 +258,12 @@ class RunProgramOpKernel : public framework::OpKernel<T> {
 
     // Debug info: scope info when run end
     VLOG(3) << framework::GenScopeTreeDebugInfo(out_scope_vec->front());
+    // Step 5. Drop all children scopes while testing.
+    if (is_test) {
+      out_scope_vec->front()->DropKids();
+    }
+    VLOG(2) << "number of sub scope after forward: "
```

Comment:
Tiny English issue:

"The number of sub scopes after forward:"

Notice the sub scope(s)

Same as the similar sentences below.

Reply:
Thx, done~